### PR TITLE
ios: improve group layout

### PR DESCRIPTION
**Pull request by epoberezkin**

### CODE REVIEW

File: apps/ios/Shared/Views/Chat/ChatItemView.swift [hunk 1]
```diff
@@ -130,60 +126,31 @@ struct ChatItemContentView<Content: View>: View {
...
-            return nil
-        }
+    private var membersConnectedText: String? {
+        var ns = chatModel.getConnectedMemberNames(chatItem)
+        return ns.count > 3
+            ? "\(ns[0]), \(ns[1]) and \(ns.count - 2) other members connected"
+            : ns.count == 3
+            ? "\(ns[0] + ", " + ns[1]) and \(ns[2]) connected"
+            : ns.count == 2
+            ? "\(ns[0]) and \(ns[1]) connected"
```
Comment (spaced4ndy) [inline apps/ios/Shared/Views/Chat/ChatItemView.swift:144]: why these shouldn't be NSLocalizedString?

File: apps/ios/Shared/Views/Chat/ChatItem/FramedItemView.swift [hunk 2]
```diff
@@ -232,17 +231,27 @@ struct FramedItemView: View {
...
+        }
+        .padding(.top, 6)
         .padding(.horizontal, 12)
     }
     
+    private func ciQuotedMsgTextView(_ qi: CIQuote, lines: Int) -> some View {
+        MsgContentView(text: qi.text, formattedText: qi.formattedText)
+        .lineLimit(lines)
+        .font(.subheadline)
+        .padding(.bottom, 6)
```
Comment (spaced4ndy) [inline apps/ios/Shared/Views/Chat/ChatItem/FramedItemView.swift:252]: formatting seems to be wrong

File: apps/ios/Shared/Views/Chat/ChatView.swift [hunk 3]
```diff
@@ -430,63 +430,64 @@ struct ChatView: View {
...
-                    .padding(.leading, 8)
-                    .environmentObject(chat)
+                    .padding(.top, 5)
+                    .padding(.trailing)
+                    .padding(.leading, 12)
+                } else {
+                    chatItemWithMenu(ci, maxWidth)
+                    .padding(.top, 5)
+                    .padding(.trailing)
+                    .padding(.leading, memberImageSize + 8 + 12)
```
Comment (spaced4ndy) [inline apps/ios/Shared/Views/Chat/ChatView.swift:464]: same

File: apps/ios/Shared/Views/Chat/ChatItemView.swift (hunk 1 above)
Comment (epoberezkin) [reply apps/ios/Shared/Views/Chat/ChatItemView.swift:144]: the question is why the should be. These are simpler.

**Pull request merged**